### PR TITLE
GameDB: add International Cue Club 2 and burnout3  demo and telly addics 

### PR DESCRIPTION
**Pull request by Mrlinkwii**
This adds  burnout3 demo , International Cue Club 2 and Telly Addicts  to  Game DB

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -6505,6 +6505,10 @@ Serial = SLED-52488
 Name   = The Suffering [Demo]
 Region = PAL-E
 ---------------------------------------------
+Serial = SLED-52597
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Might be worth to check if the crc id is different, and if that needs to be added to crc database as well for hw renderer.

Can you tell me the crc id of the demo so I can check?

Comment (Mrlinkwii) [reply bin/GameIndex.dbf]: image


this is what pcsx2  says

Comment (Mrlinkwii) [reply bin/GameIndex.dbf]: image

Comment (lightningterror) [reply bin/GameIndex.dbf]: Thanks, it's already on the list.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -6505,6 +6505,10 @@ Serial = SLED-52488
...
+Name   = Burnout 3 -Takedown [Demo]
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Little nitpick, add a space after -

**Pull request merged**